ruleset/sca/amazon/cis_amazon_linux_2.yml

```diff
@@ -46,46 +46,14 @@ checks:
       - 'c:modprobe -n -v cramfs -> r:install /bin/true|Module cramfs not found'
       - 'not c:lsmod -> r:cramfs'
 
-# 1.1.1.2 hfs: filesystem
-  - id: 20501 
-    title: "Ensure mounting of hfs filesystems is disabled"
-    description: "The hfs filesystem type is a hierarchical filesystem that allows you to mount Mac OS filesystems."
-    rationale: "Removing support for unneeded filesystem types reduces the local attack surface of the server. If this filesystem type is not needed, disable it."
-    remediation: "Edit or create a file in the /etc/modprobe.d/directory ending in .conf. Example: vim /etc/modprobe.d/hfs.confand add the following line: install hfs /bin/true. Run the following command to unload the hfs module: rmmod hfs"
-    compliance:
-      - cis: ["1.1.1.2"]
-      - cis_csc: ["5.1"]
-      - pci_dss: ["2.2.5"]
-      - tsc: ["CC6.3"]
-    condition: all
-    rules:
-      - 'c:modprobe -n -v hfs -> r:install /bin/true|Module hfs not found'
-      - 'not c:lsmod -> r:hfs'
-
-# 1.1.1.3 hfsplus: filesystem
-  - id: 20502 
-    title: "Ensure mounting of hfsplus filesystems is disabled"
-    description: "The hfsplus filesystem type is a hierarchical filesystem designed to replace hfs that allows you to mount Mac OS filesystems."
-    rationale: "Removing support for unneeded filesystem types reduces the local attack surface of the server. If this filesystem type is not needed, disable it."
-    remediation: "Edit or create a file in the /etc/modprobe.d/directory ending in .conf. Example: vim /etc/modprobe.d/hfsplus.confand add the following line: install hfsplus /bin/true. Run the following command to unload the hfsplus module: rmmod hfsplus"
-    compliance:
-      - cis: ["1.1.1.3"]
-      - cis_csc: ["5.1"]
-      - pci_dss: ["2.2.5"]
-      - tsc: ["CC6.3"]
-    condition: all
-    rules:
-      - 'c:modprobe -n -v hfsplus -> r:install /bin/true|Module hfsplus not found'
-      - 'not c:lsmod -> r:hfsplus'
-
-# 1.1.1.4 squashfs: filesystem
-  - id: 20503 
+# 1.1.1.2 squashfs: filesystem
+  - id: 20503
```

Comment:
We are missing IDs 20501 and 20502, if we remove checks we'd have to reassign IDs